string  cmdType
float64 x
float64 y
float64 theta
float64 x_cmd
float64 y_cmd
float64 theta_cmd
